NetKAN/ContractConfiguratorRO.netkan

```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version": "v1.16",
+    "identifier": "ContractConfiguratorRO",
+    "$kref" : "#/ckan/netkan/https://raw.githubusercontent.com/KSP-RO/ContractConfigurator/master/ContractConfiguratorRO.netkan",
```

Comment:
Would you mind adding a ModuleManager depends to shut up our validator/bot?

![image](https://user-images.githubusercontent.com/1559108/174464199-1be285a5-c449-4c15-8bdb-3add0e16e3aa.png)

![image](https://user-images.githubusercontent.com/1559108/174464175-ab490050-febf-4edc-85c4-8c9cb8bc5ada.png)